task_12/gulpfile.js

```diff
@@ -0,0 +1,46 @@
+var gulp = require('gulp');
+var sass = require('gulp-sass');
+var cleanCSS = require('gulp-clean-css');
+var imagemin = require('gulp-imagemin');
+var concat = require('gulp-concat');
+var uglify = require('gulp-uglify');
+const autoprefixer = require('gulp-autoprefixer');
+
+gulp.task('sass', function () {
+    return gulp.src('src/css/*.scss')
+        .pipe(sass())
+        .pipe(cleanCSS({compatibility: 'ie8'}))
+        .pipe(autoprefixer({
+            browsers: ['last 2 versions'],
```

Comment:
It's default config. Know defaults.

Reply:
So I did not read the documentation, but the nonsense
Corrected